apps/src/templates/sectionAssessments/MultipleChoiceAssessmentsOverviewTable.jsx

```diff
@@ -32,6 +38,11 @@ const styles = {
 
 const NOT_ANSWERED = 'notAnswered';
 
+function  calculateOpacity(answered) {
+  let result =  ((answered + 10) * (9 / 10))/100;
```

Comment:
no need to save this into a variable - you should be able to just return it directly.
Also this calculation is a bit confusing, can you add a comment? Does `answered + 10 / 100` not work?